Update overview.md (#844)
ingonyama-zk/icicle: docs/docs/icicle/overview.md

- **Extensibility:** Built for seamless integration and expansion, ICICLE enables you to develop and deploy custom backends and cryptographic primitives.

## Evolution from v2 to v3

Originally focused solely on GPU acceleration, ICICLE has evolved. With the release of v3, it now supports multiple backends, making it more versatile and adaptable across diverse hardware environments. Whether you're harnessing the power of GPUs or exploring alternative compute platforms, ICICLE v3 is built to meet your needs.

## Who Uses ICICLE?

ICICLE has been successfully integrated and used by leading cryptography companies such as [Brevis](https://www.ingonyama.com/blog/icicle-case-study-accelerating-zk-proofs-with-brevis), [Gnark](https://github.com/Consensys/gnark), [Zircuit](https://www.ingonyama.com/blog/case-study-accelerating-zircuits-zero-knowledge-proofs-with-icicle), [zkWASM](https://www.ingonyama.com/blog/how-icicle-helps-grow-the-zkwasm-ecosystem), [Kroma Network](https://www.ingonyama.com/blog/icicle-case-study-accelerating-zk-proofs-with-kroma-network) and others to enhance their ZK proving pipelines.

## Don't Have Access to a GPU?

> We understand that not all developers have access to GPUs—but that shouldn’t limit your ability to build with ICICLE. Here are a few ways to get access:

Reviewer: Not having access to GPUs no longer prevents building with ICICLE since we support CPU and partially Metal.

Maybe change the phrasing here to focus on "experiencing the acceleration" that icicle gives

### Grants

At Ingonyama, we are committed to accelerating progress in ZK and cryptography. If you're an engineer, developer, or academic researcher, we invite you to check out [our grant program](https://www.ingonyama.com/post/ingonyama-research-grant-2025). We can provide access to GPUs or even fund your research.

### Google Colab

> Google Colab is a great way to start experimenting with ICICLE right away. It provides free access to NVIDIA T4 GPUs—more than sufficient for prototyping and exploring ICICLE’s capabilities.

Reviewer: Either keep it as it was or
```suggestion
Google Colab is a great way to start experimenting with ICICLE right away. It provides free access to NVIDIA T4 GPUs—sufficient enough for prototyping and exploring ICICLE’s capabilities.
```

## What Can You Do with ICICLE?

> [ICICLE](https://github.com/ingonyama-zk/icicle) can be used much like any other cryptography library—but with the added benefit of acceleration. Thanks to multiple integrations, it's already proven effective across a range of use cases:

Reviewer: ```suggestion
[ICICLE](https://github.com/ingonyama-zk/icicle) can be used much like any other cryptography library with the added benefit of acceleration. Thanks to multiple integrations, it's already proven effective across a range of use cases:
```
